lib/start-button.js

```diff
@@ -100,6 +101,15 @@ var StartButton =
 
     items.push({
       nol10n: true,
+      label: Locale.$STR("rdpInspector.menu.packetCache.label"),
+      tooltiptext: Locale.$STR("rdpInspector.menu.packetCache.tip"),
+      type: "checkbox",
+      checked: prefs.packetCacheEnabled,
+      command: this.onToggleCache.bind(this)
+    })
```

Comment:
We are now having two places in the UI where options can be set (within the console and in the Start Button). I think we should have both options available on both places (inline previews and the cache option). And the Start menu should have "Options" submenu at the top separated from other menu items by a separator.

Honza